src/python/grpcio_tests/tests/tests.json

```diff
@@ -55,6 +55,7 @@
   "unit._reconnect_test.ReconnectTest",
   "unit._resource_exhausted_test.ResourceExhaustedTest",
   "unit._rpc_test.RPCTest",
+  "unit._server_shutdown_test.ServerShutdown",
```

Comment:
Can we add this test to Bazel tests in `src/python/grpcio_tests/tests/unit/BUILD.bazel`? I tried in my local machine, it should work.

Reply:
Seems you need to add the scenario file as dependency in the same way as `exit_scenarios.py` to make Bazel happy.